identityClient/api.go

```diff
@@ -688,3 +688,23 @@ type ListRealmApplicationMappersRequest struct {
 type ListRealmApplicationMappersResponse struct {
 	ApplicationMappers []ApplicationMapper `json:"application_mappers"`
 }
+
+// SearchRealmIdentitiesResponse wraps matching identities in the given realm for the search criteria (if any)
+type SearchRealmIdentitiesResponse struct {
+	IdentitiesEmailToToznyClientAliases EmailToToznyClientAliases `json:"identities_email_to_client_aliases"`
+}
+
+// EmailToToznyClientAliases maps identity email to TozID
+type EmailToToznyClientAliases map[string]uuid.UUID
+
+// SearchRealmIdentitiesRequest wraps the search criteria to use for retrieving matching Identities from a given realm
+type SearchRealmIdentitiesRequest struct {
+	RealmName      string
+	SearchCriteria SearchCriteria `json:"search_criteria"`
+}
+
+// SearchCriteria wraps identifiers to use for retrieving matching Identities from a given realm
+type SearchCriteria struct {
+	IdentityEmails   []string `json:"identity_emails"`
+	IdentityUserName []string `json:"identity_usernames"`
```

Comment:
if its UserName, it should be user_name (I'm fine with either Username or UserName just make it a consistent conversion from PascalCase to snake_case)